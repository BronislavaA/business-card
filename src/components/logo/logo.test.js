import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './logo'

test('renders Logo component', () => {
  const component = renderer.create(
    <Logo />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});