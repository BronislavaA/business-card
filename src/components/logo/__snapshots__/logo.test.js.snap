// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Logo component 1`] = `
<div
  className="sc-bdVaJa bQioAf"
>
  <img
    alt="logo"
    src="images/logo192.png"
  />
  <h2>
    Company Name
  </h2>
</div>
`;
